nixos/modules/tasks/encrypted-devices.nix

```diff
@@ -46,6 +46,12 @@ let
           so the keyfile path should usually start with "/mnt-root/".
         '';
       };
+
+      allowDiscards = mkOption {
+        default = false;
+        type = types.bool;
+        description = "cryptsetup argument --allow-discard on open";
```

Comment:
this explains how it works but not what it does.

Reply:
Now I copied the allowDiscards description from luksroot. Thank you!